src/components/InputProblem.tsx

```diff
@@ -132,12 +134,26 @@ export const InputProblem = ({
                     <WrongAnswerIcon className={'os-mr'} />
                   </div>
                 }
-                <Field
-                name="response"
-                disabled={inputDisabled || isSubmitting}
-                autoComplete={'off'}
-                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => { clearFeedback(); void setFieldValue('response', e.target.value) }}
-                className={buildClassName(values.response, solution, inputDisabled || isSubmitting)} />
+              {
+                comparator.toLowerCase() === 'math'
+                  ? (
+                  <Field
+                    name="response"
+                    disabled={inputDisabled}
```

Comment:
I know I discussed this with @MReyna12 , but what goes wrong if we make this `disabled={inputDisabled || isSubmitting}` again?

Reply:
But there's no buggy behavior if we do that? I ask because using `inputDisabled || isSubmitting` is the more technically correct approach. For example, if you add `await new Promise(r => setTimeout(r, 10000))` to `handleSubmit()`, currently while the handler is "running" a user can continue interacting with the input which is incorrect. I vaguely recalled us discussing not including `isSubmitting` due to some issue that occurred when we did, and the "race condition" issue created being the lesser issue. However, if we don't currently see problems / have a specific reason not to, I think we should keep parity with the other code path here and use `disabled={inputDisabled || isSubmitting}` as that should address both scenarios (form handler running and user attempts exhausted).

Reply:
Discussed live but noting here: The buggy behavior I was overlooking was that with `inputDisabled || isSubmitting`, after the first attempt you couldn't modify things anymore. After discussion we decided to just make a minor change to the conditional logic in the `useEffect()` hook for `disabled` to address that.

Reply:
Per a slack call with @rnathuji, we've opted to add the following code to allow for `isSubmitting` to be added back as a value of `disabled` within `InputProblem.tsx`.

```js
if (!disabled && mathFieldCurrent.shadowRoot !== null) {
      const pointerEventSpan = mathFieldCurrent.shadowRoot.querySelector('span')
      if (pointerEventSpan !== null && pointerEventSpan !== undefined) {
        pointerEventSpan.style.pointerEvents = 'auto'
      }
    }
```